### ISSUE TITLE
Encounter block does not work after updating to 13.0.3

### ISSUE THREAD

**Issue by Kardzhilov** [javalent/initiative-tracker#261]
### Check for existing bug reports before submitting.

- [X] I searched for existing Bug Reports and found no similar reports.

### Expected Behavior

Being able to use the example 
```encounter
name: Example
creatures:
 - 3: Goblin
```

### Current behaviour

![image](https://github.com/javalent/initiative-tracker/assets/15326512/f4e57624-954f-4432-b79f-26df789c5dca)

### Reproduction

Made a new vault with only the initiative tracker plugin installed
Pasted the example,
After pasting in the example it throws the first two errors and moves the name line up so that it looks like 
```encountername: Example
creatures:
 - 3: Goblin
```
after pressing enter before name it looks like the screenshot above

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [ ] Linux
- [ ] macOS
- [X] Windows

### Obsidian Version Check

v1.5.11

### Plugin Version

13.0.3

### Confirmation

- [X] I have disabled all other plugins and the issue still persists.

**Comment by valentine195** [comment]
Are you using Fantasy Statblocks?

**Comment by Kardzhilov** [comment]
I was not, I guess the version i used before did not need it but it is required now?

**Comment by valentine195** [comment]
No, it shouldn’t be. This bug is because it isn’t properly handling FS not being there.

**Comment by Kardzhilov** [comment]
A workaround for now is to install FS then 👍
